cli/cmd/compliance_aws.go

```diff
@@ -411,13 +411,85 @@ The output from status with the --json flag can be used in the body of PATCH api
 			return nil
 		},
 	}
+
+	// complianceAwsListAccountsCmd represents the list-accounts inside the aws command
+	complianceAwsSearchCmd = &cobra.Command{
+		Use:   "search <resource_arn>",
+		Short: "Search for all known violations of a given resource arn",
+		Long:  `Search for all known violations of a given resource arn.`,
+		Args:  cobra.ExactArgs(1),
+		RunE: func(_ *cobra.Command, args []string) error {
+			cli.StartProgress(fmt.Sprintf(" searching accounts for resource %s...", args[0]))
+			var (
+				now                                   = time.Now().UTC()
+				before                                = now.AddDate(0, 0, -7) // last 7 days
+				awsComplianceEvaluationSearchResponse api.ComplianceEvaluationAwsResponse
+				filter                                = api.ComplianceEvaluationSearch{
+					SearchFilter: api.SearchFilter{
+						Filters: []api.Filter{{
+							Expression: "eq",
+							Field:      "resource",
+							Value:      args[0],
+						}},
+						TimeFilter: &api.TimeFilter{
+							StartTime: &before,
+							EndTime:   &now,
+						},
+					},
+					Dataset: api.AwsComplianceEvaluationDataset,
+				}
+			)
+			err := cli.LwApi.V2.ComplianceEvaluations.Search(&awsComplianceEvaluationSearchResponse, filter)
+			cli.StopProgress()
+			if err != nil {
+				return err
+			}
+
+			if len(awsComplianceEvaluationSearchResponse.Data) == 0 {
+				cli.OutputHuman("No results found.\n")
```

Comment:
I don't think results is the right work here. what is the user searching for? could resources? 🤔

Reply:
It's meant to be a  "generic" response

With `ComplianceEvaluations/search ` we can't distinguish between the resource not existing and not having violations. Both return 204 nocontent